gun.js

```diff
@@ -1,17 +1,17 @@
 ;(function(){
 
-  /* UNBUILD */
-  function USE(arg, req){
-    return req? require(arg) : arg.slice? USE[R(arg)] : function(mod, path){
-      arg(mod = {exports: {}});
-      USE[R(path)] = mod.exports;
-    }
-    function R(p){
-      return p.split('/').slice(-1).toString().replace('.js','');
-    }
-  }
-  if(typeof module !== "undefined"){ var MODULE = module }
-  /* UNBUILD */
+	/* UNBUILD */
+	function USE(arg, req){
+		return req? require(arg) : arg.slice? USE[R(arg)] : function(mod, path){
+			arg(mod = {exports: {}});
+			USE[R(path)] = mod.exports;
+		}
+		function R(p){
+			return p.split('/').slice(-1).toString().replace('.js','');
+		}
+	}
+	if(typeof module !== "undefined"){ var MODULE = module }
+	/* UNBUILD */
```

Comment:
spacing changes should be done in a different PR for readability.

Reply:
Will keep in mind, thanks